From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Will McVicker <willmcvicker@google.com>
Date: Thu, 7 Jan 2021 17:14:12 -0800
Subject: ANDROID: gki_defconfig: add CONFIG_DEBUG_KINFO=y

This driver can be used to collected kernel information for the
bootloader.

Bug: 169101608
Change-Id: I372026656de027ff775e7fe24a7d558b89471885
Signed-off-by: Will McVicker <willmcvicker@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -456,6 +456,7 @@ CONFIG_UIO=y
 CONFIG_VHOST_VSOCK=y
 CONFIG_STAGING=y
 CONFIG_ASHMEM=y
+CONFIG_DEBUG_KINFO=y
 CONFIG_COMMON_CLK_SCPI=y
 # CONFIG_CLK_SUNXI is not set
 # CONFIG_SUNXI_CCU is not set
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -410,6 +410,7 @@ CONFIG_UIO=y
 CONFIG_VHOST_VSOCK=y
 CONFIG_STAGING=y
 CONFIG_ASHMEM=y
+CONFIG_DEBUG_KINFO=y
 CONFIG_REMOTEPROC=y
 CONFIG_REMOTEPROC_CDEV=y
 CONFIG_RPMSG_CHAR=y
